urdf/end_effector.xacro

```diff
@@ -2,7 +2,28 @@
 <robot xmlns:xacro="http://www.ros.org/wiki/xacro">
   <!-- TODO: add collision and inertial elements -->
```

Comment:
Can we remove this TODO, or does this still apply to the parallel gripper subassembly?